test/libsolidity/syntaxTests/types/magic_block_basefee_error.sol

```diff
@@ -0,0 +1,15 @@
+contract C {
+    function f() public view returns (uint) {
+        return block.basefee;
+    }
+    function g() public view returns (uint ret) {
+        assembly {
+            ret := basefee()
+        }
+    }
+}
+// ====
+// EVMVersion: =berlin
+// ----
+// TypeError 5921: (74-87): "basefee" is not supported by the VM version.
+// TypeError 5430: (183-190): The "basefee" instruction is only available for London-compatible VMs (you are currently compiling for "berlin").
```

Comment:
@ekpyron If we change it to `<=berlin`, then the CI would fail because of differences in error messages. Berlin replaced by "Istanbul", for example.

Reply:
We could also make it consistent with e.g. ``"\"chainid\" is not supported by the VM version."``, which doesn't have that problem...

Reply:
Ah, no, that's the other one. Well, fine in any case.

Reply:
Would be another reason for pulling in https://github.com/ethereum/solidity/pull/11763 first, though ;-).